tests/acceptance/browzine-primo-adapter-test.js

```diff
@@ -396,6 +396,18 @@ describe("BrowZine Primo Adapter >", function() {
           done();
         });
       });
+
+      it("should open a new window when a direct to pdf link is clicked", function() {
```

Comment:
I'm wondering if a comment could be helpful to add here. I could easily see myself or someone else coming in here in the future and going, "Why are we using window.open? A simple `a` tag with the `target` set appropriately should be fine, let me refactor this and remove this test."

Either giving a reason for why the test is here, or giving a reason in the code for why we're doing that, would probably be helpful to avoid an accidental regression in the future.